lib/config/config.go

```diff
@@ -19,14 +20,24 @@ func CompileAncestorBranches(branchName string) (result []string) {
 	}
 }
 
-func DeleteAncestorBranches(branchName string) {
-	removeConfigurationValue("git-town-branch." + branchName + ".ancestors")
+func DeleteAllAncestorBranches() {
+	configs := util.GetCommandOutput("git", "config", "--get-regexp", "^git-town-branch\\..*\\.ancestors$")
```

Comment:
I don't understand how that would help here. This is used for getting the configuration keys for all ancestor branches so they can be deleted